compiler/resolution/callDestructors.cpp

```diff
@@ -568,6 +568,28 @@ void ReturnByRef::transform()
   transformFunction(mFunction);
 }
 
+// Check for a =, PRIM_MOVE, or PRIM_ASSIGN
+// with a RHS that is marked with FLAG_FORMAL_TEMP_OUT_CALLSITE.
+// This reflects the writeback pattern added at the callsite
+// for out/inout formals (see wrappers.cpp).
+static bool isFormalTmpWriteback(Expr* e) {
+  if (CallExpr* call = toCallExpr(e)) {
+    if (call->isNamedAstr(astrSassign) ||
+        call->isPrimitive(PRIM_MOVE) ||
+        call->isPrimitive(PRIM_ASSIGN)) {
+      int nActuals = call->numActuals();
+      if (nActuals >= 2) {
```

Comment:
Not related to this PR, but do we ever add stuff to these types of calls where it can have 3 or more arguments?

Reply:
I am not sure but I think some of the operator changes can have dtMethodToken passed to things like `=`. That might just be for the formals though. I was trying to make it as robust as possible.